src/components/biggive-carousel/biggive-carousel.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-carousel',
+  styleUrl: 'biggive-carousel.scss',
+  shadow: true,
+})
+export class BiggiveCarousel {
+  @Element() host: HTMLBiggiveCarouselElement;
+
+  @Prop() spaceBelow: spacingOption = 4;
+
+  @Prop() columnCount: number = 3;
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  itemCount = 0;
+  itemWidth = 0;
+  max = 0;
+  min = 0;
+  sleeve;
+
+  componentDidRender() {
+    this.sleeve = this.host.shadowRoot?.querySelector<HTMLElement>('.sleeve');
+    let children: Array<any> = Array.from(this.host.children);
+
+    this.itemCount = children.length;
+
+    if (children.length > 0 && this.sleeve) {
+      this.itemWidth = children[0].offsetWidth;
```

Comment:
How do we know that the width of every item will be the same as the width of the first one?

Reply:
The CCS defines the width of each item, which are all set to the same.